libraries/AP_Baro/AP_Baro_SPL06.cpp

```diff
@@ -107,6 +107,16 @@ AP_Baro_Backend *AP_Baro_SPL06::probe(AP_Baro &baro,
     return sensor;
 }
 
+static int32_t get_twos_complement(uint32_t raw, uint8_t length)
```

Comment:
would be nice to have a comment explaining what this fn is

Reply:
"with thanks to betaflight" would be nice, and maybe move to AP_Math? I think we do this in several places

Reply:
Done